rocky/katalogus/views/mixins.py

```diff
@@ -83,9 +84,10 @@ def run_normalizer(self, normalizer: KATalogusNormalizer, raw_data: RawData) ->
         normalizer_task = NormalizerTask(
             id=uuid4(), normalizer=Normalizer(id=normalizer.id, version=None), raw_data=raw_data
         )
-
+        queue_name = f"normalizer-{self.organization.code}"
```

Comment:
Ideally, I'd give the organisatoin code to the push_task  method, and have tha tmethod use the item and given organisaton code to figure out the queue name